src/components/HomeInfo.vue

```diff
@@ -35,24 +41,53 @@ export default {
 .row.homeInfo
     .col.q-pa-md
       .row.items-center
-        .col-12.items-center
-          .column.items-center.text-subtitle2 TLOS Price
-        .col-12
-          .column.items-center.text-h6.text-weight-bold
-            | $ {{ tlosPrice }}
+        .col-2
+        .col-2.items-center
```

Comment:
`.items-center` seems to actually not have any styles associated with it, going by the devtools styles panel. despite the class being already used on line 43 :facepalm: 
this will center the icons vertically if that's what you wanted to do, looks good to me either way
 ```pug
 .col-2.u-flex--center
```

Reply:
yeah was just copied from the existing code, I think it isn't the only one that has it, will look

Reply:
fixed